Modules/ThirdParty/OpenJPEG/src/openjpeg/src/lib/openjp2/CMakeLists.txt

```diff
@@ -171,7 +190,10 @@ endif()
 #################################################################################
 set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
 
+#[[ -- ITK
 option(OPJ_USE_THREAD "Build with thread/mutex support " ON)
+# -- ITK]]
+set(OPJ_USE_THREAD OFF)
```

Comment:
Why do we turn this off?

Reply:
This appear to be an option to support multithreaded decoding. Our ImageIO does not use this feature, and didn't think it was work the effort to investigate further.

Reply:
It should be easy to flip or expose if we change our mind later.